docker-compose.yml

```diff
@@ -9,31 +9,28 @@ services:
 #    volumes: this persists the db code locally
 #      - ./db/data/:/var/lib/postgresql/data
     environment:
-      POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: 1234  # To be moved to environment file
-      POSTGRES_DATABASE: clinical_trials_status
-    expose:
-      - 5432
-    ports:
-      - 5432:5432
-    networks:
-      - psql-pgql
+      - POSTGRES_USER
+      - POSTGRES_PASSWORD
+      - POSTGRES_DATABASE
   graphql:
     container_name: pgql
     restart: always
     image: graphile/postgraphile
     depends_on:
       - db
-    environment:
-      DATABASE_URL: postgres://postgres:1234@db:5432/clinical_trials_status # To be moved to environment file
-    expose:
-      - 5000
     ports:
-      - 5000:5000
-    command: ["postgraphile", "--cors", "--connection", $DATABASE_URL, "--host", "0.0.0.0", "--port", "5000", "--schema", "trials_status_schema"]
+      - ${GRAPHQL_PORT}
+    command: ["postgraphile", "--cors", "--connection", "postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DATABASE}", "--host", "0.0.0.0", "--port", "${GRAPHQL_PORT}", "--schema", "trials_status_schema"]
     links:
       - db
-    networks:
-      - psql-pgql
-networks:
-  psql-pgql:
+  scraper:
+    container_name: scraper
+    # image: dstuck/clinical-trials-api-scraper:latest
+    build: ../clinical-trials-api-scraper
```

Comment:
so is this using a local build now for testing? That's a good idea

Reply:
Yeah! It was honestly accidental, something I set up for local dev. If you're happy with it though, sounds good!